lib/helpers.js

```diff
@@ -455,6 +455,8 @@ const extractMatchingPermissions = function (dumpsysOutput, groupNames, grantedS
  * @typedef {Object} InstallOptions
  * @property {boolean} allowTestPackages [false] - Set to true in order to allow test
  *                                                 packages installation.
+ * @property {boolean} allowDowngrade [false] - Set to true in order to allow
```

Comment:
this option only works for debuggable packages. 
also, package downgrade could potentially corrupt the data.

What is your reason to have it?

Reply:
@mykola-mokhnach, we have an accessibility tool (Accessibility Insights) that will be auto-installing an APK onto an Android device. We have scenarios where we'll need to downgrade our service version in response to customer actions. We were hoping to use the downgrade flag to explicitly handle this scenario. 

I was unaware of the restriction that this only works with debuggable packages--is this documented somewhere?

Reply:
yes, this is stated in both the adb source code and in the `adb help` output:

```
     -d: allow version code downgrade (debuggable packages only)
```

Reply:
Thanks, @mykola-mokhnach. I'll go ahead and close this out since it doesn't seem like it would be generally useful in a production environment.